src/mini/template.html

```diff
@@ -34,10 +34,30 @@
           'Move inside simple <template>.'));
       }
       // bootstrap the template if it has not already been
-      if (this._template && !this._template.content && HTMLTemplateElement.bootstrap) {
-        HTMLTemplateElement.decorate(this._template);
-        // recurse if necessary
-        HTMLTemplateElement.bootstrap(this._template.content);
+      if (this._template) {
```

Comment:
While here, would be nice to document that this is "when element templates are in the main document (e.g. vulcanized) they are processed before the template polyfill runs"